tools/dbus_ignorelist

```diff
@@ -2035,6 +2035,7 @@ org.freedesktop.systemd1.Swap.WatchdogSignal
 org.freedesktop.systemd1.Swap.What
 org.freedesktop.systemd1.Swap.WorkingDirectory
 org.freedesktop.systemd1.Timer.AccuracyUSec
+org.freedesktop.systemd1.Timer.DeferReactivation
```

Comment:
This list is for properties that have been around long enough and thus require no version info. Please add proper version info for new props and drop this